### PR TITLE
style(dashboard): overall improvements

### PR DESCRIPTION
**Pull request by benstov**
Screenshot 2019-06-06 at 15 49 32

### CODE REVIEW

**Comment by edvald** [comment]
One aesthetic comment: The top part of each module card feels quite crowded. Specifically, The description text is a bit close to the module name. I'd also suggest making the description less visually prominent (i.e. smaller/lighter), because it's kinda detail info imo.

File: garden-service/src/garden.ts [hunk 1]
```diff
@@ -700,6 +700,7 @@ export class Garden {
       providers: await this.resolveProviders(),
       variables: this.variables,
       moduleConfigs: sortBy(await this.resolveModuleConfigs(), "name"),
+      path: this.projectRoot,
```
Comment (eysi09) [inline garden-service/src/garden.ts]: I would re-name this variable to `projectRoot`. Path is way too generic.

Also, we've talked about separating commits `garden-service` and commits to the `dashboard`. In particular, `dashboard: overall improvements` doesn't describe this.

File: garden-service/test/unit/src/commands/get/get-config.ts [hunk 2]
```diff
@@ -27,6 +27,7 @@ describe("GetConfigCommand", () => {
       providers,
       variables: garden.variables,
       moduleConfigs: sortBy(await garden.resolveModuleConfigs(), "name"),
+      path: garden.projectRoot,
```
Comment (eysi09) [inline garden-service/test/unit/src/commands/get/get-config.ts]: Re-name to `projectRoot`.

File: dashboard/src/components/info-card.tsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+  &:empty
+{
+    display:none;
+}
+`
+
+type StateContainerProps = {
+  state: string,
+}
+const StateContainer = styled.div<StateContainerProps>`
```
[the remark this answers is not in the document]
Comment (benstov) [reply dashboard/src/components/info-card.tsx]: file is redundant, it's here by mistake, i renamed this file to entity-card.

File: dashboard/src/components/info-card.tsx [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+`
+
+const Name = styled.div`
+  font-size: 0.9375rem;
+  font-weight: 500;
+  color: rgba(0, 0, 0, .87);
+  padding-top: 0.125rem;
+`
+
+type EntityType = "service" | "test" | "task"
```
[the remark this answers is not in the document]
Comment (benstov) [reply dashboard/src/components/info-card.tsx]: file is redundant, it's here by mistake, i renamed this file to entity-card.

File: dashboard/src/components/info-card.tsx [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { ReactNode } from "react"
+import styled from "@emotion/styled"
+import { Entity } from "../containers/overview"
+import { colors } from "../styles/variables"
+import { Facebook } from "react-content-loader"
```
[the remark this answers is not in the document]
Comment (benstov) [reply dashboard/src/components/info-card.tsx]: Changed to ContentLoader

File: dashboard/src/components/module.tsx [hunk 6]
```diff
@@ -182,66 +159,59 @@ export default ({
   return (
```
[the remark this answers is not in the document]
Comment (benstov) [reply dashboard/src/components/module.tsx]: will be split up in normalisation refactor

**Comment by eysi09** [comment]
LGTM. @benstov, did you address those UI comments @edvald made?

**Comment by benstov** [comment]
> One aesthetic comment: The top part of each module card feels quite crowded. Specifically, The description text is a bit close to the module name. I'd also suggest making the description less visually prominent (i.e. smaller/lighter), because it's kinda detail info imo.

Screenshot 2019-06-12 at 12 52 27

**Comment by edvald** [comment]
That's perfect, LGTM!

**Pull request merged**